pkg/cmd/environments.go

```diff
@@ -57,7 +57,7 @@ var environmentsGetCmd = &cobra.Command{
 }
 
 func init() {
-	environmentsGetCmd.Flags().String("project", "", "output json")
+	environmentsGetCmd.Flags().StringP("project", "p", "", "output json")
```

Comment:
shouldn't the description be "doppler project"?

Reply:
good catch